src/common/components/searchInput/SearchInput.tsx

```diff
@@ -28,12 +28,8 @@ const SearchInput: React.FC<SearchInputProps> = ({
       className={classNames(className, {
         [styles.hideLabel]: hideLabel,
       })}
-      clearButtonAriaLabel={
-        clearButtonAriaLabel || getValue(t('common.clear'), undefined)
```

Comment:
getValue can be removed from imports 👀

Reply:
Good finding. I removed getValue from imports. Also remove getValue wrapping from all the tFunction calls.